ToNote/Views/MainView.xaml.cs

```diff
@@ -11,5 +11,23 @@ public MainView()
         {
             InitializeComponent();
         }
+
+        protected void Minimize_Window(object sender, RoutedEventArgs e)
+        {
+            if (this.WindowState == WindowState.Normal)
+                this.WindowState = WindowState.Minimized;
+        }
+
+        protected void Maximize_Window(object sender, RoutedEventArgs e)
+        {
+            if (this.WindowState == WindowState.Normal)
+                this.WindowState = WindowState.Maximized;
+        }
+
+        protected void Close_Window(object sender, RoutedEventArgs e)
```

Comment:
This is also a duplicate of Close_Window in BaseDialogWindow. Try to avoid this.